prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -285,4 +305,41 @@ public void verify() {
             }
         }
     }
+
+    private void getListOfProfiles() throws Exception {
+        console.println(CliMessages.MESSAGES.availableProfiles());
+        Set<String> featurePackNames = KnownFeaturePacks.getNames();
+        final ChannelPrinter channelPrinter = new ChannelPrinter(console);
+        for (String profile : featurePackNames){
+            console.println("----------");
+            console.println(CliMessages.MESSAGES.getProfile() +profile+ "\n"+CliMessages.MESSAGES.subscribedChannels());
+            for(Channel channel: getChannels(featurePackNames)){
+                channelPrinter.print(channel);
+            }
+            console.println(CliMessages.MESSAGES.includedFeaturePacks());
+            for(String featurePacks: featurePackNames){
+                FeaturePackLocation loc = getFpl(KnownFeaturePacks.getByName(featurePacks));
+                console.println("# "+loc.toString());
+            }
+
+        }
+    }
+
+    private List<Channel> getChannels(Set<String> featurePackNames) throws Exception {
+        List<Channel> channels = new ArrayList<>(List.of());
+        for (String channelNames : featurePackNames) {
+            KnownFeaturePack knownFeaturePack = KnownFeaturePacks.getByName(channelNames);
+            channels.addAll(knownFeaturePack.getChannels());
+        }
+        return channels;
+    }
+
+    private FeaturePackLocation getFpl(KnownFeaturePack knownFeaturePack) throws Exception {
+        GalleonProvisioningConfig config = GalleonUtils.loadProvisioningConfig(knownFeaturePack.getGalleonConfiguration());
+        if (config.getFeaturePackDeps().isEmpty()) {
+            throw new ProvisioningException("At least one feature pack location must be specified in the provisioning configuration");
+        }
+        return config.getFeaturePackDeps().iterator().next().getLocation();
```

Comment:
The provisioning configuration can contain multiple feature packs. In such case all of them should be printed not just the first one